pkg/loader/compose/compose.go

```diff
@@ -358,6 +358,11 @@ func (c *Compose) LoadFile(files []string) (kobject.KomposeObject, error) {
 			}
 		}
 
+		if len(serviceConfig.Port) == 0 && composeServiceConfig.Labels["kompose.service.type"] == "NodePort" || composeServiceConfig.Labels["kompose.service.type"] == "LoadBalancer" {
+			log.Fatal("Failed to proceed as Nodeport or loadbalancer is defined without ports")
```

Comment:
I think @kadel made it clear that we should fail here https://github.com/kubernetes-incubator/kompose/issues/522#issuecomment-297451523

>yes, that makes sense, we should fail, as it doesn't make sense to use NodePort or LoadBalancer without port defined.